kf-kcc-admins.members.txt

```diff
@@ -8,4 +8,6 @@ google-kubeflow-admins@kubeflow.org
 ricliu@google.com
 rleidle@google.com
 scottleehello@gmail.com
-yangpa@google.com
\ No newline at end of file
+yangpa@google.com
```

Comment:
Please alphabetize and are these Google accounts?

Reply:
Changed to use my google account.